tests/test_src/test_nuketesting/_cli/test_run_pytest_bootstrapped.py

```diff
@@ -0,0 +1,65 @@
+from __future__ import annotations
+
+from unittest.mock import MagicMock, call, patch
+
+import pytest
+from nuketesting._cli.run_pytest_bootstrapped import BootstrapException, _run_tests
+
+
+@patch("nuketesting._cli.run_pytest_bootstrapped.Path.is_dir", return_value=True)
+@patch("pytest.main", return_value=0)
+@pytest.mark.parametrize(
+    "test_directories",
+    [
+        ["first/dir", "second/dir"],
+        ["first/dir", "second/dir", "third/directory"],
+    ],
+)
+def test__run_tests_path_insertion(is_dir_mock, pytest_mock: MagicMock, test_directories: list[str]) -> None:
+    """Test path insertion to add both nuketesting directory and pytest."""
+    test_directories_combined = ":".join(test_directories)
+
+    with patch("nuketesting._cli.run_pytest_bootstrapped.sys") as sys_mock:
+        _run_tests(test_directories_combined, "", [])
+
+    for test_directory in test_directories:
+        expected_call = call(test_directory)
+        assert expected_call in sys_mock.path.append.call_args_list
+
+
+@patch("nuketesting._cli.run_pytest_bootstrapped.sys")
+@pytest.mark.parametrize(
+    ("test_directory", "test_pytest_args", "expected_arguments"),
```

Comment:
The `test_directory` can be removed from the parametrization. It's allways the same.

Reply:
I do like it for having a concise way to see what all arguments are. Also if needed to add more in the future.